plugins/coverage_reports/src/openjdk/codetools/jcov/report/jcov/JCovCoverageComparison.java

```diff
@@ -99,9 +99,9 @@ public JCovCoverageComparison(DataRoot oldCoverage, SourceHierarchy oldSource,
                         String id = className + "#" + methodName;
                         var isOldMethod = oldSource == null || oldMethodsCache.containsKey(id);
                         DataMethod oldMethod = isOldMethod ? oldMethodsCache.get(id) : null;
-                        var newLineCoverage = new MethodCoverage(newMethod, true).getLineCoverage();
+                        var newLineCoverage = new MethodCoverage(newMethod, false).getLineCoverage();
```

Comment:
@lkuskov Regarding this and similar changes in the files.
The countBlocks parameter (https://github.com/openjdk/jcov/blob/a48fd09dbc21607414c4018d7028809024532e2a/src/classes/com/sun/tdk/jcov/report/MethodCoverage.java#L59) was set to "true" everywhere in the coverage reports plugin. However, in the next constructor it is interpreted as "javapCoverage" (https://github.com/openjdk/jcov/blob/a48fd09dbc21607414c4018d7028809024532e2a/src/classes/com/sun/tdk/jcov/report/MethodCoverage.java#L68C54-L68C63). Is counting blocks equivalent to generating Java reports, or how do I interpret this?
I am guessing this is the outcome from your recent Java report mixup.
I have changed all the values to false because what I need in the reports is line coverage.
Can you advise?